lib/validations/manifests.js

```diff
@@ -57,10 +56,15 @@ const Manifests = function() {
                                sequence_bytes,
                                new Buffer(sfPublicKey).toJSON().data,
                                [master_public_bytes.length],
-                               master_public_bytes,
-                               new Buffer(sfSigningPubKey).toJSON().data,
-                               [ephemeral_public_bytes.length],
-                               ephemeral_public_bytes)
+                               master_public_bytes)
+
+    if (manifest.ephemeral_public_key) {
```

Comment:
does this need to be deployed in the validations ETL?

Reply:
Yes, in order to properly handle revocation manifests